tremor-pipeline/Cargo.toml

```diff
@@ -32,6 +32,8 @@ tremor-value = { version = "0.12.4", path = "../tremor-value" }
 url = "2.2.2"
 value-trait = "0.4"
 window = { git = "https://github.com/tremor-rs/window.git", tag = "v0.1.1" }
+anyhow = "1.0.58"
```

Comment:
Are those two unused dependencies, dev dependencies or actual runtime dependencies? I did not see them in the code.

Reply:
Sorry but I didn't understand a comment.
If I delete **anyhow** from t**remor_pipeline/Cargo.toml** I would get an error in /**tremor-pipeline/src/lib.rs**  because the appender return a **anyhow::Result<()>**

Reply:
That answers the question :) so it's a full dependency! all good then :D